src/bin/owi.ml

```diff
@@ -39,6 +39,21 @@ let files =
   let f = existing_non_dir_file in
   Cmdliner.Arg.(value & pos_all f [] (info [] ~doc))
 
+let sourcefile ty =
+  let doc = Fmt.str "%s source file" ty in
+  let f = existing_non_dir_file in
+  Cmdliner.Arg.(required & pos 0 (some f) None (info [] ~doc))
+
+let outfile ty =
+  let doc = Fmt.str "%s" ty in
```

Comment:
I believe the documentation should rather be something like `Write output to file.`